source/1_engine/rex_engine/include/rex_engine/platform.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
```

Comment:
Not sure this is the correct location for this file unless it would be used for other purposes than the assert system I don't think this should be here.